src/pages/MyPosts.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import { useLoaderData, useOutletContext } from "react-router-dom";
+import { getUserPosts } from "../api/services/posts";
+import Post from "../components/post/Post";
+import { getUsername } from "../api/services/auth/auth";
+
+export const myPostsLoader = async ( username ) => {
+  const posts = await getUserPosts(username);
+  if (posts) {
```

Comment:
Du kan da bare bruge den samme loader som du bruger i home til get poasts i stedet for at lave det samme igen <3

Reply:
Hvilket betyder at den her loader godt kan fjernes og så du bare bruger sætter den anden loader på det her også, kan bedre forklare hvis du siger til når du skal lave det